doc/user-guides/external-api.md

```diff
@@ -0,0 +1,176 @@
+# Using Gateway API and Kuadrant with APIs outside of the cluster
+
+
+### Overview
+
+In some cases, the application and API endpoints are exposed in a host external to the cluster where you are a running Gateway API and Kuadrant but you do not want it accessible directly via the public internet. If you want to have external traffic come into a Gateway API defined Gateway and protected by Kuadrant policies first being proxied to the existing legacy endpoints, this guide will give you some example of how to achieve this.
+
+
+### What we will do
+- Have an API in a private location become accessible via a public hostname
+- Setup a gateway and HTTPRoute to expose this private API via our new Gateway on a (public) domain.
+- proxy valid requests through to our back-end API service
+- Add auth and rate limiting and TLS to our public Gateway to protect it
+
+
+
+### Pre Requisites
+
+- [Kuadrant and Gateway API installed (with Istio as the gateway provider)](https://docs.kuadrant.io/latest/kuadrant-operator/doc/install/install-kubernetes/) 
```

Comment:
Would this also work on openshift?
If yes, there's a few ways users could have got here (kube install, openshift install, olm install)

Reply:
so remove the link?

Reply:
Yeah, I think that's fine.
Mention that those things are installed is sufficient. We don't have 1 entry point guide for install to link to.